akvo/rsr/static/scripts-src/project-main/project-main-partners.jsx

```diff
@@ -121,16 +124,18 @@ function renderPartnersTab() {
                         }
                     }
 
+                    var id = partner[0].organisation.id;
+
                     return (
-                        <div className='row verticalPadding projectPartners'>
+                        <div className='row verticalPadding projectPartners' key={id}>
                             <div className="col-sm-2 img">
-                                <a href={'/en/organisation/' + partner[0].organisation.id + '/'}>
+                                <a href={'/en/organisation/' + id + '/'}>
```

Comment:
Nitpick: I think we should remove /en here. I know it currently doesn't matter, but one could conceive of a future where we show different logos depending n language choice. Same goes for the name below.